packages/twenty-front/src/modules/object-metadata/utils/formatFieldMetadataItemsAsFilterDefinitions.ts

```diff
@@ -71,5 +72,7 @@ export const formatFieldMetadataItemAsFilterDefinition = ({
                     ? 'RELATION'
                     : field.type === FieldMetadataType.Select
                       ? 'SELECT'
-                      : 'TEXT',
+                      : field.type === FieldMetadataType.Address
+                        ? 'ADDRESS'
+                        : 'TEXT',
```

Comment:
Fell like this ternary is becoming quite big, we should split that into an anonymous function above with a `switch...case`